docs/manual/webhooks/datadog-run-job.md

```diff
@@ -0,0 +1,60 @@
+# DataDog (Enterprise)
+
```

Comment:
Use this code to add the "Enterprise" block that is on the other Enterprise feature pages:
::: enterprise
:::

Reply:
Included now! @fdevans